install-deps.sh

```diff
@@ -390,7 +390,7 @@ else
                 ;;
         esac
         munge_ceph_spec_in $with_seastar $for_make_check $DIR/ceph.spec
-        $SUDO $yumdnf install -y \*rpm-macros
```

Comment:
Hmm, this could break Fedora (or any other than current rhel 7 rpm builds). What about keeping the original line and adding `$SUDO $yumdnf install -y python36-devel || :` before that line?

Reply:
@b-ranto how about putting
```sh
$SUDO $yumdnf install -y python3-devel
```
in the `fedora)` branch above.
and
```sh
$SUDO $yumdnf install -y python36-devel
```
in the `centos|rhel|ol|virtuozzo)` branch above?

as "python-rpm-macros" is required by python3-devel in fc30, see https://fedora.pkgs.org/30/fedora-x86_64/python-rpm-macros-3-42.fc30.noarch.rpm.html

Reply:
I would still vote for keeping the original line (I mean, are we sure, we don't need any other than python (s)rpm-macros for the build?) and probably adding `$SUDO $yumdnf install -y python36-devel` for rhel/centos 7 as it is a rhel/centos 7-only hack. In rhel/centos 8, we already have a native python3 so we won't need that line, there.

Reply:
> are we sure, we don't need any other than python (s)rpm-macros for the build?

i think we need other rpm macros, for instance use use "fedora" and "rhel". but i don't want to use "*rpm-macros" to address this issue. i'd prefer install only the subset of packages we want.

Reply:
I understand the desire but these are generally tiny packages so they don't pollute the system much and installing even all of them looks like the safest option to me.

Reply:
@b-ranto updated and repushed.